### ISSUE TITLE
False Positive on Yields/Disable check on a line

### ISSUE THREAD

**Issue by baluyotraf** [jsh9/pydoclint#40]
Hello. Thanks for this library. It's very useful.

We're getting false positive on the yield checks. This happens when there's a sub function inside a function that uses yields. Our code looks something like this.

```
def func(arg: List[int]) -> str:
    """Converts int to string

    Args: 
        arg (List[int]): Number to convert

    Returns:
        str: String equivalent
    """

    def _convert(num: List[int]) -> Iterable[str]:
        for i in num:
            yield str(num)

    return ",".join(_convert(arg))
```

We get an error like:  106: DOC402: Function `func` has "yield" statements, but the docstring does not have a "Yields" section

As a work around, is there a way to disable the check per line?

**Comment by jsh9** [comment]
Thanks for reporting this!  It is indeed a bug.  Let me fix it.

**Comment by jsh9** [comment]
Hi @baluyotraf , I've fixed this issue as well as the false positive detection of "return" and "raise" statements.

The latest release (0.0.15) contains this fix.

Thank you again for submitting this issue!
